taxi/urls.py

```diff
@@ -16,12 +22,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
```

Comment:
Consider adding a trailing slash to the 'manufacturers/<int:pk>/delete' URL pattern to maintain consistency with other URL patterns. Consistent use of trailing slashes can help avoid confusion and potential routing issues.